### ISSUE TITLE
Scholar lost ability to find attached PDFs

### ISSUE THREAD

**Issue by ThomasJorna** [pubpub/pubpub#1193]
_Before filing your bug report, please search existing issues to make sure a similar one doesn't already exist. If one does, please leave a comment or reaction in support of the issue rather than creating a new one._

_To fill out the report, please leave the below headers (**) in tact, and fill out each section. If a section isn't applicable, leave it blank or write n/a under it. Please do not delete any template sections._

**What went wrong, step-by-step?**
1. Upload article
2. Wait week multiple weeks/months
3. Scholar does not find PDF
...

**What did you expect to happen?**
For scholar to find the PDF, as it was able to do so in the past.
This might be related to #1192 but is a separate issue that I've tried to address on it's own without much luck.

I think there are three reasons for why this might happen, one of which not much can be done about
1. As mentioned, #1192 might be the culprit. The one recently published PDF that has been found by scholar is the one for which I incorrectly set the collection metadata resulting in an empty citation_journal_title field. Not sure how it is related.
2. PubPub changed the citation_pdf_url field from a direct link to a /download/pdf relative link somewhere in September, before which there were no issues with finding the pdf. I do know that Scholar prefers direct links, so this might be the issue, and the other link is hidden behind a javascript dropdown menu.
3. Us changing from jtrialerror.pubpub.org to archive.jtrialerror.com might have been either
  a) confusing for Google
  b) our own jtrialerror.com might not be in such good standing for scholar as pubpub.org (very plausible)
  c) the PDF is hosted at assets.pubpub.org. Google explicitly mentions that the pdfs should be hosted on the same domain to be linked to the HTML version because of security reasons, so this might be the case, if not for both Google finding one of our PDFs and Google consistently finding HDSR pdfs. 
4. Changing the PDFs around too often. This might very well be so as we do do that a bit too often, however I don't think this prohibits Google from at least finding one version.
5. Google doesn't like our new PDF format. The editorial article it picked up has a markedly different markup than the rest, as it lacks an abstract. I think this unlikely to be the culprit however, as a) the pdf of our own editorial (https://doi.org/10.36850/ed1) is not found and has (almost) the same markup, b) our pdfs are not very different from before and c) when uploaded to university repositories they are picked up just fine.

I tried the following with little to no succes:
1. Insert the PDF as a file in the pub. Does not seem to have done anything, as the article found for https://archive.jtrialerror.com/pub/failfast/release/3 is not the inserted one.
2. Insert a link to pub/XXX/download/pdf in the pub
3. Insert a link to assets.pubpub.org/fbrdgfdsg/123 in the pub to bypass the JS dropdown
4. Add links to pub/XXX/download/pdf to the sitemap on search console of our main website, which google accepted.
5. Add links to assets.pubpub.org/sdfasf/1421 to our sitemap on Google Search console, which google obviously rejected.

**What URL can we use to see the issue?**

New article where PDF has been found: https://archive.jtrialerror.com/pub/failfast/release/3

Relatively old article where no pdf has been found: https://archive.jtrialerror.com/pub/conceptionbias/release/4

New HDSR article where PDF is found immediately: https://hdsr.mitpress.mit.edu/pub/twyc748y/release/1

**What browser/operating system were you using?**

---

**Have any screenshots? (Attach them by dragging the image here)**

**Who reported this? (If not you)**

**Comment by ThomasJorna** [comment]
Update: I have currently "solved" this by putting the PDF as a download on the bottom of the page. Scholar seems to have little problem finding this, as every article is now found in addition to it's PDF.

While not an ideal solution, it works for now.

**Comment by gabestein** [comment]
I'm going to close this as a bug within PubPub, as I don't think it's related to any changes we've made on the platform. That said, I'll reach out to the Scholar team to see if they can provide more insight.

Long story short: we worked with Scholar to implement our PDF meta tagging last fall, and they should be pulling PDFs correctly. That was the reason we changed from assets.pubpub.org to a proxy, as they told us they preferred a same-domain proxy over a different-domain direct link.

My guess is that the domain change is potentially at fault, and JTOE may need to be re-indexed. But we'll re-open if they come back with more details that we need to implement on our end.

**Comment by gabestein** [comment]
I asked about this with the GS team, and got a response that it's working as expected. In summary, my understanding is that when properly implemented, GS will choose to index either full-text HTML or PDF, but typically not both. In the HDSR example, and many examples we commonly see, GS is picking up the PDF as a different version of the article entirely, rather than a PDF version of the full-text html. That was technically incorrect, and we've now fixed that with the changes we made earlier this year.
